wiki/Beatmap_Editor/Menu/en.md

```diff
@@ -1,441 +1,256 @@
+Beatmap Editor/Menu
```

Comment:
Comment: this page is very long, should you split it up into smaller subpages (for the ones talking about the menus)?